lms/djangoapps/course_api/blocks/tests/test_serializers.py

```diff
@@ -136,9 +138,6 @@ def create_serializer(self):
     def test_basic(self):
         serializer = self.create_serializer()
 
-        # verify root
-        self.assertEquals(serializer.data['root'], unicode(self.block_structure.root_block_usage_key))
```

Comment:
I replaced 'return self.topological_traversal()' with 'return self.get_block_keys()' in block_structure.py:BlockStructure
as @nasthagiri suggested. So now blocks are treated as a list and no root is assigned.